openmc/mesh.py

```diff
@@ -2869,6 +2859,23 @@ def _read_meshes(elem):
     return out
 
 
+# TODO: This should really get incorporated in lower-level calls to
+# get_all_materials, but right now it requires information from the Model object
+def _get_all_materials(model: openmc.Model) -> dict[int, openmc.Material]:
```

Comment:
I must be forgetting something - can you remind me why this function should live here in mesh.py and not over on the Model class?